pkg/services/rpcsrv/server.go

```diff
@@ -2102,7 +2101,7 @@ func (s *Server) getInvokeContractVerifyParams(reqParams params.Params) (util.Ui
 		if len(args) > 0 {
 			err := params.ExpandArrayIntoScript(bw.BinWriter, args)
 			if err != nil {
-				return util.Uint160{}, nil, nil, neorpc.NewInternalServerError(fmt.Sprintf("can't create witness invocation script: %s", err))
+				return util.Uint160{}, nil, nil, neorpc.WrapErrorWithData(neorpc.ErrInvalidVerificationFunction, fmt.Sprintf("can't create witness invocation script: %s", err))
```

Comment:
Let's keep it as `InternalServerError`, verifiation function isn't related to the script builder problem.